pulp_container/app/models.py

```diff
@@ -393,6 +393,10 @@ def redirect_to_content_app(self, url):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+        permissions = [
+            ("pull_containerdistribution", "Can pull on a container distribution"),
+            ("push_containerdistribution", "Can push into a container distribution"),
```

Comment:
Since it's a user-facing  description, can we ask any of the native speakers to review this wording?
I'm not sure. "Pull on" sounds odd to my ear.
I though that one can "pull X from a registry/repo" or "push X to a registry/repo".
Is the container distribution here used as a synonym to a registry/repo?